crates/rolldown/tests/esbuild/default/external_module_exclusion_relative_path/artifacts.snap

```diff
@@ -1,15 +1,14 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
-snapshot_kind: text
 ---
 # Assets
 
 ## index.js
 
 ```js
-import foo from "./foo.js";
-import out from "../../../out/in-out-dir.js";
-import sha256 from "../../sha256.min.js";
+import foo from "./nested/folder/foo.js";
+import out from "../out/in-out-dir.js";
+import sha256 from "./sha256.min.js";
```

Comment:
The relative path external module rendered path is not finished, i will resolve it at next.

Reply:
Can you open an issue to track it ?

Reply:
alright, since #3806 is still open.

Reply:
The rolldown resut is already same as rollup, https://stackblitz.com/edit/github-shyenfvo-h8p9hpdz?file=package.json,index.js,nested%2Ffolder%2Ftest.js,rollup.config.js,dist%2Findex.js. 

The esbuild is relative to the current working directory [ref](https://github.com/evanw/esbuild/commit/345f64c1a711e22a96eeea2aae71f7523c8a347a), but the rollup using  the common shared parent directory of all entries modules [ref](https://rollupjs.org/configuration-options/#makeabsoluteexternalsrelative).

Reply:
I intend it to follow the esbuild behavior, calculating the common dir of all modules is not a good way, it always is working directory at most cases, it is unnecessary to calculate it. I will have try to it to see the breaked the rollup tests.